testdata/baselines/reference/fourslash/syntaxandSemanticDiagnostics/getJavaScriptSyntacticDiagnostics13.baseline

```diff
@@ -0,0 +1,8 @@
+// === Syntax and Semantic Diagnostics ===
+/a.js(1,8): error TS8010: Type annotations can only be used in TypeScript files.

```

Comment:
Are these files diffable?

Reply:
Forgot to mention in my description. It's only 20 something tests, and would be annoying enough to diff because Strada distinguishes between syntactic and semantic diagnostics. I just visually diffed them.